distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -6,130 +6,149 @@
+.. Select *Advanced* Search to access a query builder for more complex queries.
+. Click the save (image:save-icon.png[save icon]) icon next to the workspace title in the upper left corner. 
 
-.Navigator Menu Options
+.Navigation Menu Options
 * *Workspaces*: View all available workspaces.
-* *Upload*: Add new resources to catalog.
-* *Open Workspaces*: [Workspace Name] (if applicable) Navigate to the [Workspace Name] workspace. Each open workspace is listed in this section of the menu.
+* *Upload*: Add new metadata and resources to the catalog.
+* *Sources*: Lists all sources and their statuses.
+* *Open Workspaces*: Lists open workspaces.
 
 .Workspace Menu Options
-* Select the *Options* button (image:options-icon.png[options icon]) next to the Workspace name.
+* To view a workspace's options from the *Workspaces* view, press the *Options* button (image:options-icon.png[options icon]) for the workspace.
 ** *Save*: Save changes to the workspace.
 ** *Run All Searches*: Start all saved searches within this workspace.
 ** *Cancel All Searches*: Cancel all running searches.
 ** *Open in New Tab*: Opens this workspace in a separate tab.
-** *View Sharing*: View and edit settings for sharing this workspace. (User must be signed in to share or view shared workspaces.)
-** *View Details*: View the current details for a cloud-based workspace (User must be signed in).
+** *View Sharing*: View and edit settings for sharing this workspace. Users must be signed in to share workspaces or view shared workspaces.
+** *View Details*: View the current details for a cloud-based workspace Users must be signed in to view workspace details.
 ** *Duplicate*: Create a copy of this workspace.
-** *Subscribe*: Receive email notifications for search results on this workspace (If email notifications have been enabled).
+** *Subscribe/Unsubscribe*: Selecting *Subscribe* will enable email notifications for search results on this workspace. Selecting *Unsubscribe* will disable email notifications for search results on this workspace.
 ** *Move to Trash*: Delete (archive) this workspace.
 
 ==== Sharing Workspaces
 
-In ${catalog-ui}, workspaces can be shared between users at different levels of access as needed.
+Workspaces can be shared between users at different levels of access as needed.
 
 .Share a Workspace
-. Select the Workspace.
-. Select the *Options* menu.
+. From the *Workspaces* view, select the *Options* menu (image:options-icon.png[options icon]) for the workspace in which sharing will be modified.
 . Select *View Sharing*.
-.. To share by user role, set the drop-down menu to *Can Access* for each desired role. All users with that role will be able to view.
-.. To share with an individual user, add their email to the email list.
+.. To share by user role, set the drop-down menu to *Can Access* for each desired role. All users with that role will be able to view the workspace.
+.. To share with an individual user, add his/her email to the email list.
```

Comment:
❓ The email will not always be the attribute required to share a workspace. It is configurable via the Admin UI in `Search UI` -> `Configurations` -> `Workspace Security`. Should we note that somewhere here in the Share a Workspace section?

Reply:
I talked with @djblue and our current implementation can only share workspaces by role and by email.

Reply:
Ah - I was thinking about creating workspaces. 👍